coeus/coeus_analysis/src/analysis/instruction_flow.rs

```diff
@@ -1734,6 +1734,16 @@ impl InstructionFlow {
                 }
                 Instruction::ArrayData(_, _) => {}
                 Instruction::SwitchData(_) => {}
+
+                Instruction::ShrIntLit8(dst, left, lit) => {
+                    b.state.registers[u8::from(dst) as usize] =
+                        &b.state.registers[u8::from(left) as usize] >> (lit as i128)
+                }
+                Instruction::UShrIntLit8(dst, left, lit) => {
+                    b.state.registers[u8::from(dst) as usize] =
+                        &b.state.registers[u8::from(left) as usize] >> (lit as i128)
```

Comment:
With the trait impl at hand we would then change this line here
```suggestion
                        &b.state.registers[u8::from(left) as usize].ushr(lit as i128)
```